assets/panel/panel.js

```diff
@@ -68,10 +63,10 @@ DebuggerPanel.prototype = {
     return { frames, selected };
   },
 
-  destroy: function () {
+  destroy: function() {
```

Comment:
DevTools ESLint style in m-c requires `function (` with the space. Should we update the debugger `.eslintrc` so that they match on this point?